lib/lambda/update/updatePackage.test.ts

```diff
@@ -0,0 +1,287 @@
+import { describe, it, expect, vi, beforeEach } from "vitest";
+import { handler } from "./updatePackage";
+import { APIGatewayEvent } from "node_modules/shared-types";
+
+import {
+  EXISTING_ITEM_ID,
+  EXISTING_ITEM_PENDING_ID,
+  CAPITATED_INITIAL_ITEM_ID,
+  CAPITATED_INITIAL_NEW_ITEM_ID,
+  WEIRD_ID,
+} from "mocks";
+vi.mock("libs/handler-lib", () => ({
+  response: vi.fn((data) => data),
+}));
+
+describe("handler", () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+    process.env.topicName = "test-topic";
+  });
+
+  it("should return 400 if event body is missing", async () => {
+    const event = {} as APIGatewayEvent;
+    const result = await handler(event);
+    const expectedResult = { statusCode: 400, body: { message: "Event body required" } };
+
+    expect(result).toStrictEqual(expectedResult);
+  });
+
+  it("should return 400 if package ID or action is not found", async () => {
+    const noActionevent = {
+      body: JSON.stringify({ packageId: "123", changeReason: "Nunya" }),
+    } as APIGatewayEvent;
+
+    const resultPackage = await handler(noActionevent);
+
+    expect(resultPackage.statusCode).toBe(400);
+    const noApackageEvent = {
```

Comment:
you should probably split these into two tests just for clarity

Reply:
split